apps/desktop-wallet/src/components/AppHeader.tsx

```diff
@@ -52,16 +52,15 @@ const AppHeader: FC<AppHeader> = ({ children, title, className, invisible }) =>
   const theme = useTheme()
   const dispatch = useAppDispatch()
   const defaultAddress = useAppSelector(selectDefaultAddress)
-  const mnemonic = useAppSelector((s) => s.activeWallet.mnemonic)
-  const isPassphraseUsed = useAppSelector((s) => s.activeWallet.passphrase)
+  const isAuthenticated = useAppSelector((s) => !!s.activeWallet.id)
```

Comment:
Since we are not storing the mnemonic anymore, we now compute if a wallet is unlocked or not by checking of there is an active wallet id in redux.

Reply:
Hm, for such a small logic (`!!`) I don't think it's necessary, but I really don't have a strong opinion on this, I can do it.

Reply:
Addressed in 2662fbd3f in PR #504